apps/api/routers/gh_actions.py

```
@@ -0,0 +1,509 @@
+    finally:
+        client.close()
+
+
+@router.post(
+    "/runs",
+    response_model=DispatchResponse,
+    dependencies=[Depends(_require_gh_monitor)],
+)
+def dispatch_run(
+    body: DispatchRequest,
+    _user: Dict[str, Any] = Depends(require_role("admin")),
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)